package.json

```diff
@@ -25,23 +25,23 @@
     "tsconfig.json"
   ],
   "dependencies": {
-    "@eslint-community/eslint-plugin-eslint-comments": "^4.4.0",
-    "@eslint/js": "^9.11.1",
-    "@typescript-eslint/utils": "^8.0.0",
-    "@vitest/eslint-plugin": "^1.1.7",
+    "@eslint-community/eslint-plugin-eslint-comments": "^4.4.1",
+    "@eslint/js": "^9.15.0",
+    "@typescript-eslint/utils": "^8.15.0",
+    "@vitest/eslint-plugin": "^1.1.10",
     "confusing-browser-globals": "^1.0.11",
     "eslint-import-resolver-typescript": "^3.6.3",
     "eslint-plugin-css-modules": "^2.11.0",
     "eslint-plugin-import": "^2.31.0",
-    "eslint-plugin-jest": "^28.2.3",
-    "eslint-plugin-jsx-a11y": "^6.10.0",
+    "eslint-plugin-jest": "^28.9.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
     "eslint-plugin-lodash": "^8.0.0",
-    "eslint-plugin-react": "^7.37.0",
+    "eslint-plugin-react": "^7.37.2",
     "eslint-plugin-react-hooks": "^5.0.0",
     "eslint-plugin-sort-destructure-keys": "^2.0.0",
-    "eslint-plugin-unicorn": "^56.0.0",
+    "eslint-plugin-unicorn": "^56.0.1",
     "eslint-plugin-unused-imports": "^4.1.4",
-    "typescript-eslint": "^8.8.0"
+    "typescript-eslint": "^8.15.0"
```

Comment:
Yep, this seems to address the issue we've previously encountered in the integration!

Ref: https://github.com/typescript-eslint/typescript-eslint/issues/10338#issuecomment-2483750053